src/00/01-rfc-process/proposal.adoc

```diff
@@ -0,0 +1,280 @@
+input from the wider community. Examples of such not-impactful decisions:
+
+* Editorial changes to the standards, like section organization, grammar, and
+  spelling.
+* Details of reference implementations. I.e. how something is implemented is not
+  important as long as the standards are followed.
+
+[#design]
+== Design
+
+The sections below are duplicated in the appropriate location of the root
+`README.adoc`. Which is included as part of this RFC. Upon approval these will
+become the main body of the `README.adoc`. Additionally the following are an
+expected consequence of adoption:
+
+* Creation of status labels in https://github.com/ecostd/rfcs[RFCs repository]:
+  `rfc-active`, `rfc-needs-std-pr`, `rfc-std-pr`, `rfc-approval-requested`,
+  `rfc-approval-voting`, `rfc-approved`, `rfc-rejected`, and possibly others.
+
+* Creation of a GitHub project to automatically track status of RFCs based on
+  the status label.
+
+:leveloffset: +1
+
+[#process]
+== Process
+
+In order to add or make changes to the EcoStd standards one must get an RFC
+approved (i.e. proposed, discussed, voted on, and merged). Before following the
+steps below, make sure to read this entire process. As there are considerations
+to keep in mind that will improve the chances of your proposal being effective.
+The steps to submit an RFC are:
+
+. Fork the https://github.com/ecostd/rfcs[RFCs repository].
+
+. Copy the `src/00/00-template-adoc` (or `src/00/00-template-md` for
+  Markdown
+  footnote:gfm[GitHub Flavored Markdown (https://github.github.com/gfm/)])
+  directory to `src/00/00-sliced-bread` (change `sliced-bread` to
+  something short, but descriptive).
+
+. Write your RFC in the `proposal.adoc` (or `proposal.md` for Markdown
+  footnote:gfm[])
+  file. If you need additional files you should place them *only* in the 
+  directory. It's important, i.e. required, to properly fill in the `authors` 
+  and `email` metadata fields at the top with your information. As they are 
+  used for proper license attribution. Consult the
+  https://docs.asciidoctor.org/asciidoc/latest/[Asciidoctor documentation],
+  or on how to do that.
+
+. Submit a pull request. In addition to the RFC itself you should provide a
+  brief description in the pull request itself. Copy-pasting the RFC abstract
+  into the PR description is minimally sufficient.
+
+. Use the pull request number you now have to rename the
+  `src/00/00-my-proposal` directory to match the PR number and fill the
+  `rfcpr` field in `proposal.adoc` (or `proposal.md` for Markdown
+  footnote:gfm[]) to match.
+
+. At this time the RFC will be considered _active_ and will be labeled
+  accordingly. The RFC will receive initial feedback for which you will need to
+  respond, preferably both in the PR comments and reflected in the RFC text
+  itself.
+
+. After some time, and sufficient discussion and interest, and if this is a
+  proposal for a standard change (or addition) you wil need to write wording for
```

Comment:
```suggestion
  proposal for a standard change (or addition) you will need to write wording for
```

I don't trust myself to find all of the typos. Maybe copy/paste this into an editor with spell check and see what else it finds?